packages/enzyme-test-suite/test/ReactWrapper-spec.jsx

```diff
@@ -6026,6 +6030,385 @@ describeWithDOM('mount', () => {
       });
     });
 
+    describeIf(is('>= 16.6'), 'getDerivedStateFromError', () => {
+      describe('errors inside an error boundary', () => {
+        const errorToThrow = new EvalError('threw an error!');
+
+        const hasFragments = is('>= 16.2');
```

Comment:
this will always be true in 16.6+, so i think we can inline a `true` for this, and optimize further as a result.